File: leetcode/1easy/권민서/1961.java
import java.util.*;


class Solution {
    public boolean isPrefixString(String s, String[] words) {
        StringBuilder prefix = new StringBuilder();

        for (String word : words) {
            prefix.append(word);

            if(prefix.toString().equals(x)) {
                return true;
            }
            if(prefix.length() > s.length()) {
                return false;
            }
        }
        return false;
    }
}

File: leetcode/1easy/위성원/1961.js
/**
 * @param {string} s
 * @param {string[]} words
 * @return {boolean}
 */
var isPrefixString = function (s, words) {
  let concatS = "";
  for (const word of words) {
    concatS += word;

    if (concatS === s) {
      return true;
    }

    if (concatS.length >= s.length) {
      return false;
    }
  }

  return false;
};